### DISCUSSION TITLE
Neg. Rate - Read > whats going wrong?

### DISCUSSION THREAD

**Discussion by nuki47** [jomjol/AI-on-the-edge-device#2692] [Q&A]
my first steps here....but I can't understands whats going wrong?

In the configuration (Expert view) > Check Digit Increase Consistency is true

![watermeteree](https://github.com/jomjol/AI-on-the-edge-device/assets/75117725/9ca97b5d-5d5e-4d5c-b229-8b22cef59a93)

**Comment by caco3** [comment]
I think it is unable to recognise your ROIS properly. You should improve the focus first! Its all a bit too blurry.

**Reply by caco3** [reply]
And the last Digit ROI looks like it is too far right.
If needed, you can disable the equidistance between the ROIs!
